docs/src/components/home/ProductsSwitcher.tsx

```diff
@@ -100,7 +100,7 @@ const ProductsSwitcher = ({
       aria-label="Go to core components page"
       icon={<IconImage name="product-core" />}
       name="MUI Core"
-      description="Ready-to-use foundational components, free forever."
+      description="The only React UI library you need. Includes Material UI."
```

Comment:
```suggestion
      description="The only React UI library you need. Includes Material UI."
```

I'm not sure I follow. Material UI is a react UI library. How can it be "included"?

Reply:
My understanding is that _MUI Core_ is a React UI library that includes _Material UI_ as one set of components (among others that are in the works). Is there a better term to use to describe MUI Core?

Reply:
Core is a collection of libraries.